src/index.html

```diff
@@ -11,7 +11,174 @@
+          Name:
+          <input
+            type="text"
+            name="name"
+            minlength="2"
+            maxlength="32"
+            autocomplete="off"
+            required
+          >
+        </label>
+
+        <label class="form">
+          How old are you?
+          <input
+            type="number"
+            name="age"
+            min="1"
+            max="100"
+            value="12"
+          >
+        </label>
+
+        <label class="form">
+          Full date of birth:
+          <input
+            type="date"
+            name="data"
+          >
+        </label>
+
+        <label class="form">
+          I accept the term of the agreement
+          <input
+            type="checkbox"
+            name="term"
+            required
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="groups">
+        <legend>Registration</legend>
+
+        <label class="form">
+          E-mail:
+          <input
+            type="email"
+            name="e-mail"
+            placeholder="email@example.com"
+            required
+            multiple
+          >
+        </label>
+
+        <label class="form">
+          Password:
+          <input
+            type="password"
+            name="password"
+            minlength="6"
+            maxlength="15"
+            required
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="groups">
+        <legend>An interesting fact about you!</legend>
+
+        <div class="form"> Do you love cats?
+          <label>
+            <input
+              type="radio"
+              name="cats"
+              value="y"
+            >
+            Yes
+          </label>
+
+          <label>
+            <input
+              type="radio"
+              name="cats"
+              value="n"
+            >
+            No
+          </label>
+        </div>
+
+        <label class="form">
+          What is your favorite color?
+          <input
+            type="color"
+            name="color"
+            value="#000000"
+          >
+        </label>
+
+        <label class="form">
+          What time do you go to bed?
+          <input
+            type="time"
+            name="time"
+          >
+        </label>
+
+        <label class="form">
+          What are your favorite brand of cars?
+          <select name="car" multiple>
+            <option value="bmw">BMW</option>
+            <option value="audi">Audi</option>
+            <option value="lada">Lada</option>
+          </select>
+        </label>
+
+        <label class="form">
+          How do you rate out work?
+          <input
+            type="range"
+            name="rate"
+            value="0"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="groups">
+        <legend>Additional info:</legend>
+
+        <label class="form">
+          Comments:
+          <textarea
+            name="comments">
+          </textarea>
```

Comment:
![image](https://user-images.githubusercontent.com/99331793/187885863-c24abe35-d8a1-4f80-912d-6009f29fa8cb.png)

white spaces in the text area
```suggestion
          <textarea name="comments"></textarea>
```